src/assets/sass/_global.scss

```diff
@@ -16,8 +18,19 @@ h2 {
   border-bottom: 4px solid;
   font-size: 32px;
   padding-bottom: 10px;
-  width: 350px;
   margin-bottom: 5px;
+  text-align: center;
+
```

Comment:
should this space be removed?

Reply:
@suzannahmahala what space? Are you referring to the entirety of line 23? I guess this is personal preference (not mine)